lib/salestation/app/input_verification.rb

```diff
@@ -18,7 +18,7 @@ def verify_input(schema)
                 Errors::InvalidInput.new(errors: result.errors, hints: result.hints)
               )
             end
-          elsif dry_validation_version <= Gem::Version.new('1.8')
+          elsif dry_validation_version >= Gem::Version.new('1.0')
```

Comment:
maybe `else`? because if checks `< 1.0` and this has been updated to `>= 1.0`, this means that else is never reached.